src/globals.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.png' {
```

Comment:
why do we need this? can't you just inline the png path in https://github.com/kubevirt-ui/kubevirt-plugin/pull/379/files#diff-52a107f248365357ba6e5dbb6bb3f3617058f9a112552af68f2df2d479ba0df3R57

Reply:
Typescript without this line give me this error
``` Cannot find module './migration-illustration.png' or its corresponding type declarations.```